### PR TITLE
Please review this, @casey-collab

### PR DESCRIPTION
**Pull request by BartMulder**
Please review this, @casey-collab

### CODE REVIEW

File: dutch/over-het-hek/mobiel/mobiel.md [hunk 1]
```diff
@@ -3,3 +3,4 @@ Je druk op het knopje van je mobiel, het scherm licht nog even op en...
 ...heb jij weer, 'Battery too low' en het scherm dimt, maar in het laatste schijnsel
 zie je nog net een schim vlakbij voor je langs bewegen, wat is dit nu weer??
 
+De schim vervolgt zijn weg zonder je op te merken. In der verte verschijnt een [lichtje](lichtje/lichtje).
```
Comment (casey-collab) [inline dutch/over-het-hek/mobiel/mobiel.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by BartMulder** [comment]
Fixed and tested, all OK.

**Comment by 13rac1** [comment]
@BartMulder You have the oldest open PR in this project. I'm going to see what the bot says.

@casey-collab

**Comment by casey-collab** [comment]
Looks great! :smiley_cat:

**Pull request merged**